src/pages/Detail/components/DetailAside/HeartToggleButton.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { AiOutlineHeart, AiFillHeart } from 'react-icons/ai';
+import DetailButton from '../DetailButtons/DetailButton';
+
+import styled from 'styled-components';
+
+export default function HeartToggleButton({ click, isClicked }) {
```

Comment:
```suggestion
export default function HeartToggleButton({ onClick, isClicked }) {
```
특별한 이유 없다면 인터페이스의 형태는 유지시켜주는게 사용 측에서 덜 헷갈릴것 같습니다

Reply:
오 이벤트네이밍과 동일해도 괜찮은 것이었군요! 수정하였습니다 감사합니당!